<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="qForwardButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Vorwärts</string>
    </property>
   </widget>
   <widget class="QPushButton" name="qLeftButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Links</string>
    </property>
   </widget>
   <widget class="QPushButton" name="qBackwardButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Rückwärts</string>
    </property>
   </widget>
   <widget class="QPushButton" name="qRightButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Rechts</string>
    </property>
   </widget>
   <widget class="QPushButton" name="qStopButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>341</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="qLinieFolgenButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>300</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Linie folgen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="qDriveCircleButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>360</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Kreis fahren</string>
    </property>
   </widget>
   <widget class="QPushButton" name="qActivateSensorsButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>420</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Sensoren aktivieren</string>
    </property>
   </widget>
   <widget class="QPushButton" name="qFollowCorridorButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>470</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Korridor folgen</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>300</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Radius [cm]</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>510</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Geschwindigkeit [%]</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="qRadiusSpinBox">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>290</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="qSpeedSpinBox">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>480</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
